### PR TITLE
adds article voting

### PR DESCRIPTION
**Pull request by kamisera**
Fixed logic relating to vote toggling
Style changes in next PR

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -47,6 +63,40 @@ const Article = ({ currentArticle }) => {
               {currentArticle.comment_count === 1 ? " comment" : " comments"}
             </Link>
           </p>
+          <p className="article-info">
+            <span className="votes-label">Votes:</span> {currentVotes}
+          </p>
+          <div className="article-vote-buttons">
```
Comment (MrMcSnail) [inline src/components/Article.jsx:69]: I really like the styling on these vote buttons. Make sure you end up with a consistant UI though, as your vote buttons are different on the comment components.

**Review by MrMcSnail — CHANGES_REQUESTED** [review]
YOu votes appear to toggle for the user. But they affect the database incrementally so the votes the user  thinks they have cast my not be reflected accurately in the database in the event that a user presses the vote buttons multiple times. 
Rework the logic and resubmit a PR

File: src/components/Article.jsx (hunk 1 above)
Comment (kamisera) [reply src/components/Article.jsx:69]: Will make them consistent in the next PR 👍

**Comment by kamisera** [comment]
> YOu votes appear to toggle for the user. But they affect the database incrementally so the votes the user thinks they have cast my not be reflected accurately in the database in the event that a user presses the vote buttons multiple times.
> Rework the logic and resubmit a PR

I have reimplemented the logic and it should be working as expected. A user can upvote or downvote, and if they toggle their vote the API will be updated too. If they click upvote and then click downvote (or vice versa) the api will be sent a request for double the amount i.e. upvote is +1, then if they downvote, it should go back to 0, then -1 (so sending a API request with -2)

A user can vote again if they reload the page though since there is no vote attribution in the back-end

**Review by MKD01 — APPROVED** [review]
Great work, im happy for you to merge this in 👍

**Pull request merged**